translations/messages+intl-icu.de.yml

```
@@ -10,7 +10,7 @@ access.token: "Zugangscode"
 access.token.send: "Zugangscode versenden"
 access.token.copied: "Zugangscode wurde kopiert."
 accessibility: "Barrierefreiheit"
-accessibility.barrier.report: "Barriere melden"
+accessibility.report_barrier: "Barriere melden"
 accessibility.explanation: "Erklärung zur Barrierefreiheit"
```

nitpick: alphabetic order of keys

```suggestion
accessibility.explanation: "Erklärung zur Barrierefreiheit"
accessibility.report_barrier: "Barriere melden"
```